scripts/get_plane_model.py

```diff
@@ -0,0 +1,28 @@
+def get_plane_model(md_name,lk_name,vs_name,xc, yc, w,l):
+    
+    plane_str = """
+    <model name='{}'>
+      <static>1</static>
+      <link name='{}'>
+        <visual name='{}'>
+          <pose>{:.5f} {:.5f} 0 0 0</pose>
```

Comment:
```suggestion
          <pose>{:.5f} {:.5f} 0 0 0 0</pose>
```
This should contain 6 numbers corresponding to x y z r p y

Reply:
Corrected